tests/test_openwrt_15.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests for `openwrt_luci_rpc` package."""
+
+import unittest
+import config
```

Comment:
`No module named 'config'`

Reply:
That's odd, is included for me in python 3.7.3

I will switch back to environment variables.

Reply:
I'm also on 3.7.3. 

Did you forget to commit a new file? config does not exist in the repo.

Reply:
Oh yeah! I had a config.py file with the credentials to access my router (username and password) and I obviously added that to gitignore

I rolled back to env variables so it should not be a problem anymore